dashboard/db/schema.rb

```diff
@@ -742,6 +742,8 @@
     t.integer  "script_id",       limit: 4
     t.integer  "level_source_id", limit: 4
     t.boolean  "submitted"
+    t.boolean  "view_answers"
```

Comment:
Maybe viewable_answers?

That way you can say that the field indicates whether the activity has viewable_answers or not.

@ashercodeorg or @aoby might have ideas too.